client/src/common/definitions/union/IPartySettings.ts

```diff
@@ -0,0 +1,3 @@
+export interface IPartySettings {
+  sortByItemScore: boolean; // unused currently
```

Comment:
why is it here then? ;)

Reply:
True! Will remove all unused party settings files